packages/components/forms/stories/FormControl.stories.tsx

```diff
@@ -13,12 +21,74 @@ export default {
 
 export const Basic = (args: FormControlInternalProps) => {
   return (
-    <FormControl {...args}>
-      <FormControl.Label isRequired>Name</FormControl.Label>
-      <TextInput label="input" />
+    <>
+      <FormControl {...args}>
+        <FormControl.Label isRequired>Name</FormControl.Label>
+        <TextInput label="input" />
```

Comment:
I thought the label, in this case, would be provided by `FormControl.Label`
why do we need to pass label here?

Reply:
Right now it's enforced by TextInput API, but @burakukula is going to remove it.